packages/babel-types/test/cloning.js

```diff
@@ -72,4 +72,21 @@ describe("cloneNode", function() {
       node.declarations[0].id.typeAnnotation,
     );
   });
+
+  it("should support deep cloning without loc", function() {
+    const node = t.variableDeclaration("let", [
+      t.variableDeclarator({
+        ...t.identifier("value"),
+        typeAnnotation: t.anyTypeAnnotation(),
+      }),
+    ]);
```

Comment:
I think we need to add `loc` objects to some of the nodes to test that loc isn't copied.

Reply:
Thanks for the clarification, just question how I can add `loc` objects to some of the nodes and witch nodes we should add loc objects to?

Reply:
Just doing `node.loc = {}` is enough to test that `loc` isn't copied over. To test the `deep` part, we need to set a loc object on both the root node and some deeper node, like the `value` identifier.

Reply:
I added `loc` for `value` the deeper node.